### ISSUE TITLE
Cursor in replay browser does not reset correctly

### ISSUE THREAD

**Issue by Endaris** [panel-attack/panel-game#247]
Reported by mato [here](https://discord.com/channels/196674532491132937/362093490538020864/1037175287609315338), can reproduce this.

1. Watch a vs replay that is further down the list in a set.
2. Play a game of vs self and finish it so a replay is generated.
3. Enter replay browser

-> Cursor is in the same position where the replay watched earlier was and pressing A will crash the game.

**Comment by z26** [comment]
Easy fix, added an if statement to detect if the replay folder changed.

```
function set_replay_browser_path(path)
  local new_path = string.sub(path, (string.len(replay_browser.base_path) + 1)) .. "/"
  if replay_browser.current_path ~= new_path then
    replay_browser.cursor_pos = 0
    replay_browser.current_path = new_path
  end
end
```

**Comment by Endaris** [comment]
Fixed with 69cd4ba54f4ea31aafd2d1992bd70c58dee458ba
